tests/macsec/profile.json

```diff
@@ -5,31 +5,31 @@
         "primary_cak": "0123456789ABCDEF0123456789ABCDEF",
         "primary_ckn": "6162636465666768696A6B6C6D6E6F70",
         "policy": "security",
-        "send_sci": "false"
+        "send_sci": "true"
```

Comment:
Don't need to change this field, because we have profile "128_SCI" to cover this case.

Reply:
done